config/redirects.yml

```diff
@@ -350,6 +350,7 @@ redirects:
   "/blog/how-to-find-school-experience": "/is-teaching-right-for-me/get-school-experience"
   "/blog/what-to-expect-on-your-teacher-training": "/train-to-be-a-teacher/initial-teacher-training"
   "/blog/school-experience-helped-me-decide-to-switch": "/is-teaching-right-for-me/get-school-experience"
+  "/blog/managing-behaviour-in-the-classroom": "/is-teaching-right-for-me/behaviour-management"
 
   # Blog tags
```

Comment:
Need to add in two further redirects for the blog tag pages, now that you are removing the tags

FROM `/blog/tag/behaviour` TO new behaviour page
FROM `/blog/tag/classroom-concerns` TO new behaviour page